pytorch_lightning/callbacks/lr_monitor.py

```diff
@@ -67,6 +69,21 @@ def configure_optimizer(self):
             }
             return [optimizer], [lr_scheduler]
 
+    Example::
+
+        def configure_optimizer(self):
+            optimizer = torch.optim.SGD(
+                [
+                    {
+                        'params': [p for p in self.parameters()],
+                        'name': 'my_parameter_group_name'
+                    }
+                ],
+                lr=0.1
```

Comment:
nit: less use less whitespace for the docs

```suggestion
                [{
                    'params': [p for p in self.parameters()],
                    'name': 'my_parameter_group_name'
                }],
                lr=0.1
```